Document/0x05a-Platform-Overview.md

```diff
@@ -139,22 +138,26 @@ drwxr-xr-x   6 sven  staff   204B Dec  5 16:17 assets
 drwxr-xr-x   3 sven  staff   102B Dec  5 16:18 lib
 drwxr-xr-x  27 sven  staff   918B Dec  5 16:17 res
 -rw-r--r--   1 sven  staff   241K Dec  5 14:45 resources.arsc
-```
 
-* **AndroidManifest.xml**: Contains the definition of app’s package name, target and min API version, app configuration,  components, user-granted permissions, etc.
-* **META-INF**: This folder contains metadata of the app:
-   * MANIFEST.MF: Stores hashes of app resources.
-   * CERT.RSA: The certificate(s) of the app.
-   * CERT.SF: The list of resources and SHA-1 digest of the corresponding lines in the MANIFEST.MF file.
-* **assets**: A directory containing app assets (files used within the Android App like XML, Java Script or pictures) which can be retrieved by the AssetManager.
-* **classes.dex**: The classes compiled in the DEX file format understandable by the Dalvik virtual machine/Android Runtime. DEX is Java Byte Code for Dalvik Virtual Machine. It is optimized for running on small devices.
-* **lib**: A directory containing libraries that are part of the APK, for example 3rd party libraries that are not part of the Android SDK.
-* **res**: A directory containing resources not compiled into resources.arsc.
-* **resources.arsc**: A file containing precompiled resources, such as XML files for the layout.
+    AndroidManifest.xml: Contains the definition of app’s package name, target and min API version, app configuration, components, user-granted permissions, etc.
+    
+    META-INF: This folder contains metadata of the app:
+        MANIFEST.MF: Stores hashes of app resources.
+        CERT.RSA: The certificate(s) of the app.
+        CERT.SF: The list of resources and SHA-1 digest of the corresponding lines in the MANIFEST.MF file.
+    
+    assets: A directory containing app assets (files used within the Android App like XML, Java Script or pictures) which can be retrieved by the AssetManager.
+    
+    classes.dex: The classes compiled in the DEX file format understandable by the Dalvik virtual machine/Android Runtime. DEX is Java Byte Code for Dalvik Virtual Machine. It is optimized for running on small devices.
+    
+    lib: A directory containing libraries that are part of the APK, for example 3rd party libraries that are not part of the Android SDK.
+    
+    res: A directory containing resources not compiled into resources.arsc.
+    
+    resources.arsc: A file containing precompiled resources, such as XML files for the layout.

```

Comment:
By identing this is becomes a code block. I think the previous formatting was better.